conans/client/cmd/new_ci.py

```diff
@@ -247,7 +248,7 @@ def ci_get_files(name, version, user, channel, visual_versions, linux_gcc_versio
     if gitlab_clang_versions is True:
         gitlab_clang_versions = clang_versions
     if osx_clang_versions is True:
-        osx_clang_versions = ["7.3", "8.0", "8.1", "9.0"]
+        osx_clang_versions = ["7.3", "8.1", "9.0"]
```

Comment:
Just to make sure: apple-clang was not modified in latest release, we don't have info about ABI compatibility for apple-clang, the version scheme (4.0.1, 5.0.X) is just for clang